sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/CliSuite.scala

```diff
@@ -573,4 +573,14 @@ class CliSuite extends SparkFunSuite with BeforeAndAfterAll with Logging {
     // the date formatter for `java.sql.LocalDate` must output negative years with sign.
     runCliWithin(1.minute)("SELECT MAKE_DATE(-44, 3, 15);" -> "-0044-03-15")
   }
+
+  test("SPARK-33100: Ignore the content inside bracketed comment and ignore the comment without" +
+    " content.") {
+    runCliWithin(1.minute)("/* SELECT 1;*/ SELECT 1;" -> "1" )
+    runCliWithin(1.minute)(";;/* SELECT 1;*/ SELECT 1;" -> "1" )
+    runCliWithin(1.minute)("/* SELECT 1;*/;; SELECT 1;" -> "1" )
+    runCliWithin(1.minute)("SELECT '1'; -- SELECT '1';" -> "")
+    runCliWithin(1.minute)("SELECT '1'; /* SELECT '1';*/" -> "")
+    runCliWithin(1.minute)("/* SELECT 1;*/;; SELECT 1; /* SELECT '1';*/" -> "" )
```

Comment:
What do we try to capture here?

Reply:
It seems this case is redundant, will remove it.